helpers/fuzzybool_test.go

```diff
@@ -0,0 +1,114 @@
+package helpers_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/qlik-oss/gopherciser/helpers"
+)
+
+func TestFuzzyBool_UnmarshalJSON(t *testing.T) {
+	tests := []struct {
+		name    string
+		args    []byte
+		wantErr bool
+		wantSb  helpers.FuzzyBool
+	}{
+		{
+			name:    "Test empty",
+			args:    []byte(``),
+			wantErr: false,
+			wantSb:  true,
+		},
+		{
+			name:    "Test null",
+			args:    []byte(`null`),
+			wantErr: false,
+			wantSb:  true,
+		},
```

Comment:
Are these expected to be `true`? Seems counter intuitive.

Reply:
yes as per comment in begining of file, default is true. Mostly used for "conditions" etc where all values expect explicitly false is true.

Reply:
i.e. null = no condition has been set and can thus be considered fulfilled